website/docs/props/tab.md

```diff
@@ -50,6 +50,16 @@ Disable the indicator below
 
 ---
 
+### `indicatorStyle`
+
+Additional styling for tab component (optional)
```

Comment:
Tab indicator not component

Reply:
yes indicator is the proper word here, Thanks